masterfabric-mobile/swift-camp
# CL - HStack Viper Implementation

## 📋 PR Description
This PR introduces the implementation of an HStack view using the VIPER architecture pattern. The key components (View, Presenter, Wireframe) have been defined and structured to follow a modular approach for better separation of concerns and scalability.

---

## ✅ Checklist


---

## 🛠 Steps to Test

1. Launch the application.
2. Navigate to HomeView.
3. Click the "Go to HStack" button.
4. Verify that the HStackView screen appears.
5.  Click the back button and verify that it navigates back to HomeView.

---

## 🔗 Related Links

- Documentation: https://medium.com/@dickareynaldisiahaan/viper-architecture-in-swift-building-robust-ios-apps-ed30d673d459

---

### 📷 Screenshots (Optional)
 
| HomeView Navigation Button    | HStack Interface |
|-------------------------------|--------------------------|
| ![Simulator Screenshot - iPhone 16 Pro - 2024-12-28 at 19 49 47](https://github.com/user-attachments/assets/86e0e6d5-2456-45eb-ab91-8ef96f7f72e4) |  ![Simulator Screenshot - iPhone 16 Pro - 2024-12-28 at 19 49 51](https://github.com/user-attachments/assets/7d763dc8-2c6e-466f-ae68-61e9c221bfaf) |
